rplugin/python3/nvim-typescript/__init__.py

```diff
@@ -213,24 +220,33 @@ def tssig(self):
                 'echohl WarningMsg | echo "TS: Server is not Running" | echohl None')
 
     # Various Auto Commands
+    @neovim.autocmd('CursorHold', pattern='*.ts,*.tsx,*.js,*.jsx')
```

Comment:
Lets do this in plugin/nvim-typescript.vim

Something like this should work fine

```vim
  if get(g:, 'nvim_typescript#type_info_on_hold', 1)
    if get(g:, 'nvim_typescript#javascript_support', 1)
       autocmd CursorHold *.ts,*.tsx,*.js,*.jsx TSType
    endif
     autocmd CursorHold *.ts,*.tsx TSType
  endif
```